netgrasp/netgrasp.py

```diff
@@ -437,7 +437,8 @@ def log_event(mid, iid, did, rid, event, have_lock = False):
         ng.debugger.debug("entering log_event(%s, %s, %s, %s, %s, %s)", (mid, iid, did, rid, event, have_lock))
 
         # Only log events for which there are subscribers.
-        if ((ng.email["enabled"] and event in ng.email["alerts"]) or (ng.notification["enabled"] and event in ng.notification["alerts"])):
+        if (ng.email["enabled"] and event in ng.email["alerts"]) \
+            or (ng.notification["enabled"] and event in ng.notification["alerts"]):
```

Comment:
That's a great point and why I thought a bit before making the change. I want you to know that the last thing I want to do is "pep8 you", i.e. send you a bunch pep8 changes. :) That's not my intention, although it may appear that way or actually be the case for some changes I submit (if I run across them while fixing something else). I'd like to try and explain my motivation for that.

My motivation is to use the code on my home network and have a tool for network visibility and to learn/contribute to the project when I have time. So with regards to the contribute part, my overall motivation is code readability and maintenance. I'll try to explain more below.

One of the things I've read and learned is code is read a lot more than it's written. Especially when you're coming into a new code base/project. So I've been reading a lot of the code and trying to process how it all fits together. When I'm doing that and come across a long line (>120 characters), I need to scroll horizontally to read it and it takes me out of context a little bit. I can't see all of the code on the page.

In regards to the code snippet above, I bounced around on what the best way to read it would be. What would be the least error-prone and bug-inducing way to reason about it? My opinion is for it to all be visible without scrolling (<120 characters) and manually wrapped so the operator starts the line, i.e. so it's not "dangling" at the end of the line. I think it's easier to read and maintain and less bug-prone if it's written the way I submitted it above.

I don't have a link to share or remember where I've seen this, but I'll link to a good pep8 presentation by Raymond Hettinger. I try to read/watch all of his content I find. Really good stuff. [Raymond Hettinger - Beyond PEP 8 -- Best practices for beautiful intelligible code - PyCon 2015](https://www.youtube.com/watch?v=wf-BqAjZb8M)

Sorry for the long comment, but I wanted to try and explain. In a nutshell, for me it's about readability and being able to understand the code and spot bugs, which also folds into maintainability.